portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,376 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400&display=swap');
+* {
+  box-sizing: border-box;
 }
 
-#greeting-container {
+body, html {
```

Comment:
Some thoughts on whether to apply styles to body or html https://css-tricks.com/html-vs-body-in-css/

Reply:
read article, how does this look? (I'm not sure why I have to specify height: 100% in html, but it seems like the 100% within body takes 100% of 100% from <html>. If I don't specify that in html, seems like it's taking 100% of 0. Does that sound right?) 

`html {
  background-color: #0C0C0C;
  height: 100%;
}

body {
  margin: 0;
  height: 100%;
  font-family: 'Comfortaa', cursive;
}`

Reply:
sg!